backend/dto/posts.js

```diff
@@ -0,0 +1,22 @@
+import models from '../database/models'
+
+/* eslint-disable import/prefer-default-export */
+export const convertToPostDto = async (post) => {
+    const author = await models.users.findByPk(post.author)
+    const children = await models.posts.findAll({
+        attributes: ['id'],
+        where: { parent: post.id },
+    })
+
+    return {
+        id: post.id,
+        content: post.text_content,
+        author,
+        parent: post.parent,
+        children: children.map((child) => child.id),
+        usersLiked: 0, // work in progress.
+        usersShared: 0, // work in progress.
+        timestamp: post.updatedAt,
```

Comment:
front-end expects the timestamp to be returned as a unix timestamp, updatedAt is of type date so it should be converted first

Reply:
I think I fixed it.